packages/shared/wallet/src/hooks/use-intercom-update.tsx

```diff
@@ -1,16 +1,63 @@
 import { useNotionalContext } from '@notional-finance/notionable-hooks';
-import { SupportedNetworks } from '@notional-finance/util';
+import {
+  getFromLocalStorage,
+  setInLocalStorage,
+  SupportedNetworks,
+} from '@notional-finance/util';
 import { useIntercom } from 'react-use-intercom';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
+
+const DeBankAPIKey = process.env['NX_DE_BANK_API_KEY'] as string | undefined;
```

Comment:
it should be "NX_DEBANK_API_KEY", debank is one word

Reply:
Typescript freaked out about this being DEBANK. I had it that originally. When I separated the words it was fine